shark/iree_utils.py

```diff
@@ -15,6 +15,7 @@
 import iree.runtime as ireert
 import iree.runtime.scripts.iree_benchmark_module as benchmark_module
 import iree.compiler as ireec
+import iree.compiler.tflite as ireec_tflite
```

Comment:
could we check if the module is available before importing ? We can clean up in the next commit but would be good to keep the core runtime pieces independent of the importer utils.

Reply:
we don't have this for mac yet😅

Reply:
OK. But, this module will be deleted, it's never used.